get_all_manifests.sh

```
@@ -4,28 +4,55 @@ set -e
+    # Supports three ref-name formats:
+    # 1. "branch" - tracks latest commit on branch (e.g., main)
+    # 2. "tag" - immutable reference (e.g., v1.0.0)
+    # 3. "branch@commit-sha" - tracks branch but pinned to specific commit (e.g., main@a1b2c3d4)
+    declare -A COMPONENT_MANIFESTS=(
+        ["dashboard"]="red-hat-data-services:odh-dashboard:rhoai-3.0:manifests"
+        ["workbenches/kf-notebook-controller"]="red-hat-data-services:kubeflow:rhoai-3.0:components/notebook-controller/config"
+        ["workbenches/odh-notebook-controller"]="red-hat-data-services:kubeflow:rhoai-3.0:components/odh-notebook-controller/config"
+        ["workbenches/notebooks"]="red-hat-data-services:notebooks:rhoai-3.0:manifests"
+        ["kserve"]="red-hat-data-services:kserve:rhoai-3.0:config"
+        ["ray"]="red-hat-data-services:kuberay:rhoai-3.0:ray-operator/config"
+        ["trustyai"]="red-hat-data-services:trustyai-service-operator:rhoai-3.0:config
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)